amazon_sqs_receive_message_block.py

```diff
@@ -1,23 +1,15 @@
 from nio.util.discovery import discoverable
-from nio.properties import StringProperty, IntProperty, ListProperty
+from nio.properties import StringProperty, IntProperty
+from nio.signal.base import Signal
+
 from .amazon_sqs_base_block import SQSBase
 
 
 @discoverable
 class SQSReceiveMessage(SQSBase):
     """Send message over Amazon SQS
```

Comment:
Also looks like they wouldn't have to specify a message body here as this is receive